Cake.Recipe/Content/testing.cake

```diff
@@ -136,12 +134,56 @@ BuildParameters.Tasks.TestFixieTask = Task("Test-Fixie")
     })
 );
 
-BuildParameters.Tasks.TestTask = Task("Test")
-    .IsDependentOn("Test-NUnit")
-    .IsDependentOn("Test-xUnit")
-    .IsDependentOn("Test-MSTest")
-    .IsDependentOn("Test-VSTest")
-    .IsDependentOn("Test-Fixie")
-    .Does(() =>
-{
+BuildParameters.Tasks.DotNetCoreTestTask = Task("DotNetCore-Test")
+    .IsDependentOn("Install-OpenCover")
+    .Does(() => {
+    // Since we need to run dotnet test for each unit test library
+    // and merge the coverage result at each run, we need to delete
+    // the existing file in case it exists from a previous run.
+    if (FileExists(BuildParameters.Paths.Files.TestCoverageOutputFilePath))
+    {
+        DeleteFile(BuildParameters.Paths.Files.TestCoverageOutputFilePath);
```

Comment:
oh right, completely forgot.
Based it on my own build script for Cake.Transifex, which doesn't clean the coverage file before the unit testing task.